hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/wipe/WipeTranslator.java

```diff
@@ -61,8 +64,27 @@ public boolean matches(@NonNull final HtsCallAttempt attempt) {
      */
     @Override
     public HtsCall callFrom(@NonNull final HtsCallAttempt attempt) {
+        final var body = bodyForClassic(attempt);
+        final var isFungibleWipe = body.tokenWipeOrThrow().serialNumbers().isEmpty();
```

Comment:
oh here's the same thing only this time it's serial numbers.  at least _this_ seems directly tied to NFT or not, more than "has a memo".  Still, not the best way to express semantics.